features/stellar.yaml

```diff
@@ -0,0 +1,7 @@
+logging: 30
+migrate_from_0_3_2: false
+project_name: shopozor
+stellar_url: postgres://postgres:@postgres:5432/stellar_data
+tracked_databases:
+- postgres
+url: postgres://postgres:@postgres:5432/template1
```

Comment:
what is this `template1` database?

Reply:
it's a file generated automatically, it's internal stuff to stellar